backend/amd64/proc.ml

```diff
@@ -23,7 +23,7 @@ open Cmm
 open Reg
 open Mach
 
-let simd_regalloc_disabled () = not (Language_extension.is_enabled SIMD)
+let simd_regalloc_disabled () = not (Language_extension.is_enabled SIMD || !simd_regalloc)
```

Comment:
I am not sure it is the right condition;
if the langage extension is on, disabled
will be false independently of the arch-
specific parameter.

Reply:
That is the intent - I think users should only need to enable the extension